x-pack/plugins/monitoring/public/components/logstash/pipeline_viewer/models/pipeline/if_statement.js

```diff
@@ -18,6 +20,12 @@ function makeStatementsForOutgoingVertices(outgoingVertices, statements, next, p
   });
 }
 
+function addStatementsToList(list, statements, depth, id) {
+  statements.forEach(statement => {
+    list.push(...statement.toList(depth + 1, id));
```

Comment:
This is a bit nitpicky but I don't think this function should _also_ increase the depth when adding statements to the list; that means it's doing two things but the name doesn't really reflect that. I'd increment the depth on the calling end.